src/hiero_sdk_python/utils/entity_id_helper.py

```
@@ -123,3 +125,36 @@ def format_to_string_with_checksum(shard: int, realm: int, num: int, client: "Cl
 
     base_str = format_to_string(shard, realm, num)
     return f"{base_str}-{generate_checksum(ledger_id, format_to_string(shard, realm, num))}"
+
+def perform_query_to_mirror_node(url: str, timeout: float=10) -> Dict[str, Any]:
+    """Perform a GET request to the Hedera Mirror Node REST API."""
+    print(url)
+    if not isinstance(url, str) or not url:
+        raise ValueError("url must be a non-empty string")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)